lib/internal/modules/esm/get_source.js

```diff
@@ -33,9 +37,19 @@ async function defaultGetSource(url, { format } = {}, defaultGetSource) {
       throw new ERR_INVALID_URL(url);
     }
     const { 1: base64, 2: body } = match;
-    source = Buffer.from(decodeURIComponent(body), base64 ? 'base64' : 'utf8');
+    source = BufferFrom(decodeURIComponent(body), base64 ? 'base64' : 'utf8');
+  } else if (experimentalNetworkImports && (
+    parsed.protocol === 'https:' ||
+    parsed.protocol === 'http:'
+  )) {
+    const res = await fetch(parsed, context);
+    source = await res.body;
   } else {
-    throw new ERR_INVALID_URL_SCHEME(['file', 'data']);
+    throw new ERR_UNSUPPORTED_ESM_URL_SCHEME(parsed, [
+      'file',
+      'data',
+      ...(experimentalNetworkImports ? ['https', 'http'] : []),
+    ]);
```

Comment:
```suggestion
    throw new ERR_UNSUPPORTED_ESM_URL_SCHEME(parsed, ArrayPrototypeConcat([
      'file',
      'data',
      experimentalNetworkImports ? ['https', 'http'] : [],
    ]);
```

Reply:
Didn't I do this already…? 🤔

Reply:
lol i thought so, but i figured i'd post the comment again anyways